src/Error/Error.php

```diff
@@ -19,6 +19,8 @@
  *
  * Class extends standard PHP `\Exception`, so all standard methods of base `\Exception` class
  * are available in addition to those listed below.
+ *
+ * @see \GraphQL\Tests\Error\ErrorTest
```

Comment:
I think we should not add these.

1. this is nonexistent symbol when installed via composer since we don't bundle tests
2. it's better to use `@covers` to link test to the classes it covers